### PR TITLE
Release notes for helm chart 1.12.0

### PR DESCRIPTION
**Pull request by jedcunningham**

### CODE REVIEW

**Review by hussein-awala — APPROVED** [review]

File: chart/RELEASE_NOTES.rst [hunk 1]
```diff
@@ -23,6 +23,73 @@ Run ``helm repo update`` before upgrading the chart to the latest version.
...
+New Features
+^^^^^^^^^^^^
+
+- Enable native HPA for Airflow Workers (#36174)
+- Add init container + sidecar support for Airflow Kerberos (#35548)
+
+Improvements
+^^^^^^^^^^^^
+
+- Improve PriorityClass to improve debuggability (#36365)
```
Comment (eladkal) [inline chart/RELEASE_NOTES.rst:75]:
```suggestion
- Improve PriorityClass to improve debug capabilities (#36365)
```

With this it should not require adding the word to the spelling wordlist

**Review by eladkal — APPROVED** [review]

File: chart/RELEASE_NOTES.rst (hunk 1 above)
Comment (jedcunningham) [reply chart/RELEASE_NOTES.rst:75]: Is there an issue with adding the new word?

Comment (eladkal) [reply chart/RELEASE_NOTES.rst:75]: just wondered if debuggability is a real word :)

Comment (jedcunningham) [reply chart/RELEASE_NOTES.rst:75]: Oh, yeah, it is a real word :) not as common, but real.

**Review by dirrao — APPROVED** [review]

**Comment by potiuk** [comment]
Could we include this one too please (just merged) https://github.com/apache/airflow/pull/36898

**Comment by potiuk** [comment]
I also added https://github.com/apache/airflow/pull/36930 with producing reproducible tarball (including verification process) following the reproducible release processes we have for all other releases. 

Still the helm packages are not binary-reproducible, but that would likely require another change.

**Review by Taragolis — APPROVED** [review]

**Comment by jedcunningham** [comment]
> Could we include this one too please (just merged) #36898

Yep, I was waiting on this one!

**Comment by jedcunningham** [comment]
Okay, this should be ready now :)

**Review by amoghrajesh — APPROVED** [review]
LGTM

**Review by potiuk — APPROVED** [review]

**Review by ephraimbuddy — APPROVED** [review]

**Review by hussein-awala — APPROVED** [review]

**Review by Taragolis — APPROVED** [review]

**Pull request merged**